BlueDBOnDisk/src/main/java/org/bluedb/disk/collection/CollectionMetaData.java

```diff
@@ -15,11 +17,13 @@
 	
 	private static final String FILENAME_SERIALIZED_CLASSES = "serialized_classes";
 	private static final String FILENAME_KEY_TYPE = "key_type";
+	private static final String FILENAME_VERSION = "bluedb_version";

```

Comment:
I was originally thinking that we should make BlueDBVersion an interface so that we can create new versions of it if we need to. However, this isn't the type of a collection, its just one object stored by a specific key. There is no reason that we can't create a new key and a new type of version and have some code to convert the old one if we ever needed to change it. Its nice to have the option even though we probably won't need to change it.

I think I'm good with this merge now except for one thing. The outer build.gradle contains a version. That version is what determines the version on the jar. It needs to always match the `CURRENT` version in our code. We should probably make it so you only have to update the gradle file. If you deploy or build the project it should set the `CURRENT` version in the code to match the version in the gradle file. Or something like that.